app/views/waste_carriers_engine/renew_registration_forms/new.html.erb

```diff
@@ -1,5 +1,51 @@
 <%= render("waste_carriers_engine/shared/back", back_path: back_renew_registration_forms_path(token: @renew_registration_form.token)) %>
 
 <div class="text">
-  <h1 class="heading-large">Renew registration Form - TODO</h1>
+  <%= form_for(@renew_registration_form) do |f| %>
+    <%= render("waste_carriers_engine/shared/errors", object: @renew_registration_form) %>
+
+    <h1 class="heading-large"><%= t(".heading") %></h1>
+
+    <div class="form-group <%= "form-group-error" if @renew_registration_form.errors[:temp_lookup_number].any? %>">
+      <fieldset id="temp_lookup_number">
```

Comment:
Do we actually need a fieldset here? Me and Andrew have removed a lot of them recently, as they should only be needed when there is more than one field to group together, like in the case of checkboxes or radio buttons :)

Reply:
Probably not, I think this just comes from copying and pasting! I'll update it.